src/app/routes/utils/jsonClone/index.js

```diff
@@ -0,0 +1,3 @@
+const jsonClone = object => JSON.parse(JSON.stringify(object));
```

Comment:
I like the name of this function, but should we go one step further and write a comment that makes it clear it shouldn't be used on standard objects (i.e. with non-serialisable properties)? Or that _should only be used with JSON_.